workflow-sample-fingerprinting/src/main/java/ca/on/oicr/pde/workflows/SampleFingerprintingWorkflow.java

```diff
@@ -180,7 +180,7 @@ public void buildWorkflow() {
                         + " --datadir " + this.dataDir
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " --datadir " 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)